content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -81,6 +81,14 @@ The Kubernetes API server flag `disable-admission-plugins` takes a comma-delimit
 kube-apiserver --disable-admission-plugins=PodNodeSelector,AlwaysDeny ...
 ```
 
+## What plugins are enabled by default?
+
+The default enabled admission plugins are: 
+```
+NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
+```
+The `Priority` admission plugin is enabled by default when using `--enable-admission-plugins`. If using `--admission-control` to fully specify the set of admission plugins, the `Priority` admission plugin should be added if using the `PodPriority` feature, which is enabled by default in 1.11. 
```

Comment:
--admission-control is deprecated.
Not sure we want to describe a deprecated flag in document.

Reply:
@CaoShuFeng  Simplified.